packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -59,6 +60,10 @@ export function bindPreferenceSchemaProvider(bind: interfaces.Bind): void {
     bindContributionProvider(bind, PreferenceContribution);
 }
 
+export function bindDefaultPreferenceProvider(bind: interfaces.Bind): void {
+    bind(PreferenceProvider).to(PreferenceSchemaProvider).inSingletonScope().whenTargetNamed(PreferenceScope.Default);
```

Comment:
There is already instance of `PreferenceSchemaProvider` provided via `bindPreferenceSchemaProvider`.
`bindDefaultPreferenceProvider` should not create a completely different instance for `PreferenceProvider` symbol.

You can just inject `PreferenceSchemaProvider` in `plugin-contribution-handler`:
```ts
@inject(PreferenceSchemaProvider)
private readonly preferenceSchemaProvider: PreferenceSchemaProvider;
```

Reply:
fixed
BTW is it OK for you to use get `PreferenceSchemaProvider` instance like this:
https://github.com/theia-ide/theia/blob/f82f3adce449c46b6cacd5a5f61a31c319330c82/packages/core/src/browser/frontend-application-module.ts#L197-L199

Reply:
yes, it should be alright